### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -6,10 +6,38 @@ packages:
...
   - translation:
+      key: "txt.apps.search_app.app.long_description"
+      title: "Long Description for App: Sidebar Search"
+      value: "The Search app allows you to search your Zendesk Support for tickets, comments, users, organizations and topics.
+
+
+It offers both basic and advanced search functionality so that agents are able to quickly find information to help solve tickets more efficiently. Agents can filter their search by status conditions, assignee and also by 'create' and 'update' date ranges. In addition, the Search application offers search suggestions based on the ticket's subject and custom ticket fields.
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
```
Comment (injuhwang) [inline translations/en.yml]: Shouldn't "... You agree to the..." be changed to "... you agree to the..."?

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍 for the strings

**Comment by pmgrove** [comment]
@svizzari 💇

**Pull request merged**